python/ray/state.py

```diff
@@ -1029,6 +1094,15 @@ def error_messages(self, driver_id=None):
 global_state = DeprecatedGlobalState()
 
 
+def drivers():
+    """Get a list of the drivers in the cluster.
+
+    Returns:
+        Information from the driver table.
```

Comment:
Can you add more info for this? i.e., "Dict mapping X to Y". Is the ended usage `ray.global_state.driver_table`? 

I find that the global state is really important and useful but super underdocumented and underexposed.

Reply:
I added the information here: https://github.com/ray-project/ray/pull/5076/files#diff-44655bf61dec5dbed8cd6432860f9f59R1049